docs/pages/management/join-services-to-your-cluster/aws-ec2.mdx

```diff
@@ -1,46 +1,48 @@
 ---
-title: Joining Nodes via AWS EC2 Identity Document
-description: Use the EC2 join method to add Nodes to your Teleport cluster on AWS
+title: Joining Services via AWS EC2 Identity Document
+description: Use the EC2 join method to add services to your Teleport cluster on AWS
 ---
 
 This guide will explain how to use the **EC2 join method** to configure Teleport
-Nodes and Proxy Service instances to join your Teleport cluster without sharing
-any secrets when they are running in AWS.
+processes and Proxy Service instances to join your Teleport cluster without
```

Comment:
Since we're not using "Nodes" anymore, are we differentiating "Teleport processes" from "Proxy Service", which is itself a Teleport process?

Reply:
I've condensed this to just "Teleport processes"